examples/react-rich/src/App.tsx

```diff
@@ -53,7 +54,7 @@ const removeStylesExportDOM = (
   return output;
 };
 
-const exportMap = new Map<
+const exportMap: DOMExportOutputMap = new Map<
   Klass<LexicalNode>,
   (editor: LexicalEditor, target: LexicalNode) => DOMExportOutput
 >([
```

Comment:
This type of the value can be inferred from the explicit type of the variable now, so it can be even simpler than this:

```suggestion
const exportMap: DOMExportOutputMap = new Map([
```

Reply:
Nevermind, it looks like the Klass<LexicalNode> is still required, so I'm reverting that suggestion.